workflow-templates/openapi.yml

```diff
@@ -64,7 +64,9 @@ jobs:
           npm ci
 
       - name: Set up dependencies
-        run: composer i
+        run: |
+          composer remove nextcloud/ocp --dev
```

Comment:
In this workflow also server is not cloned, so no OCP available.

Reply:
removed the change